package.json

```diff
@@ -250,7 +250,18 @@
           "description": "An array of RegExps to use as TODO for note suggestions"
         }
       }
-    }
+    },
+    "colors": [
+      {
+        "id": "codeAnnotaion.mainColor",
+        "description": "Main color.",
+        "defaults": {
+          "dark": "#FFFFFF13",
+          "light": "#0000000C",
+          "highContrast": "#FFFFFF13"
```

Comment:
I think these are fairly subtle, not in a bad way. I think the next step in this (which can be done in a different PR) is to allow configuration of the colors, as well as if the user wants or not the decorations.

Reply:
I agree, it's gonna be good idea to recall all things that can be configured and place it in extension config.